src/overlays/actors/ovl_Obj_Makekinsuta/z_obj_makekinsuta.h

```diff
@@ -11,8 +11,8 @@ typedef void (*ObjMakekinsutaActionFunc)(struct ObjMakekinsuta*, PlayState*);
 typedef struct ObjMakekinsuta {
     /* 0x0000 */ Actor actor;
     /* 0x014C */ ObjMakekinsutaActionFunc actionFunc;
-    /* 0x150  */ s16 timer;
-    /* 0x152  */ s16 unk_152;
+    /* 0x0150 */ s16 timer;
+    /* 0x0152 */ s16 unk_152;
 } ObjMakekinsuta; // size = 0x0154
```

Comment:
This should use 3 digits instead of 4 anyway.

Reply:
idk about "should" but sure I'll change it now
This isn't exactly consistent in the repo

Reply:
Usually we do it with the required number of digits' padding and no more.

Reply:
looking for `/* 0x000 ` I find 61 results,
while with (regex) `/\* 0x0.*\n\}` (matches ends of structs with a leading `/* 0x0...` comment on the last member) I find 800

looks sus, at least definitely not usual

Reply:
What Elliptic is probably trying to say is there's no need to have the extra zeroes since the struct's size is just `3 digits (without the `0x`)
At least in MM land we have been trying to avoid those extra zeroes

Reply:
I do understand the concept of minimal zero padding, just not convinced OoT even made an attempt at it

Reply:
the bootstrap script that generated our initial actor scripts used 4 0's no matter what, which is probably why most dont currently do it. we only really standardized omitting leading zeros in asset xmls afaik. but yeah, its something we can look to do in further cleanup prs